Library/Formula/llvm.rb

```diff
@@ -106,7 +139,7 @@ class Llvm < Formula
     depends_on CodesignRequirement
   end
 
-  keg_only :provided_by_osx
+  depends_on "isl" if build.with? "polly"
```

Comment:
Just to complement [my comment](https://github.com/Homebrew/homebrew/pull/46721#issuecomment-164157454): Remove this dependency.